get_all_manifests.sh

```diff
@@ -16,7 +16,7 @@ declare -A COMPONENT_MANIFESTS=(
     ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
     ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
     ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.12.0-rc0:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.12.0:config:odh-model-controller"
+    ["odh-model-controller"]="opendatahub-io:odh-model-controller:incubating:config:odh-model-controller"
     ["kserve"]="opendatahub-io:kserve:release-v0.12.1:config:kserve"
```

Comment:
please update this one to `release-v0.14`

Reply:
this is being used for odh nightly, right?

Reply:
yes